content/docs/tutorials/interactivity/wrapping-up@es.md

```diff
@@ -0,0 +1,19 @@
+---
+$title: Terminando
+$order: 4
+---
+
+¡Felicitaciones! Acaba de crear una página de detalles de productos de comercio electrónico interactivo con tamaños y precios variables para cada SKU, obtenido a petición de un punto final JSON remoto.
+
+Has aprendido a:
+
+- Use [`<amp-state>`](/docs/reference/components/amp-bind.html#state) para obtener la información del producto (color, tamaño, disponibilidad).
+- Use [`<amp-bind>`](/docs/reference/components/amp-bind.html) para crear interacciones entre elementos.
+- ¡Crea una experiencia de usuario rápida y rica con AMP!
+
+Si te quedaste atascado, mira el archivo [`static/final.html`](https://github.com/googlecodelabs/advanced-interactivity-in-amp/blob/master/static/final.html) para la solución completa.
+
+
+## Próximos pasos
+
+Obtenga más información sobre el poder y la flexibilidad de crear páginas de AMP interactivas con la documentación de [`<amp-bind>`](https://www.ampproject.org/docs/reference/components/amp-bind.html) y con otros [componentes AMP dinámicos](https://www.ampproject.org/docs/reference/components#dynamic-content).
```

Comment:
Change the link to `/es/docs/reference/components.html#conteido-dinámico`

Reply:
Done